crates/frui_widgets/src/flex/mod.rs

```diff
@@ -7,6 +7,30 @@ pub use row::*;
 pub mod center;
 pub mod column;
 pub mod row;
+pub mod stack;
+pub mod alignment;
+
+#[derive(Debug, Clone, Copy, Default)]
+pub struct BoxLayoutData {
+    offset: Offset
+}
+
+pub trait LayoutData<T = BoxLayoutData> {
+    fn layout_data(&self) -> &T;
+
+    fn layout_data_mut(&mut self) -> &mut T;
+}
+
+impl LayoutData for BoxLayoutData {
+    fn layout_data(&self) -> &BoxLayoutData {
+        self
+    }
+
+    fn layout_data_mut(&mut self) -> &mut BoxLayoutData {
+        self
+    }
+}
```

Comment:
`LayoutData` isn't used anywhere, is it?

Reply:
I want to use this as `ParentData` in the future, only those parent widgets that won't change layout info could not need this, but I think a parent widget without changing layout is useless.

Reply:
Ok